src/Microsoft.DotNet.Cli.Utils/ProjectContextExtensions.cs

```diff
@@ -63,5 +69,118 @@ public static string GetDefaultRootOutputPath(ProjectContext context, string cur
 
             return rootOutputPath;
         }
+        
+        internal static void MakeRunnable(this ProjectContext context, string outputPath, string configuration)
+        {
+            context
+                .ProjectFile
+                .Files
+                .GetCopyToOutputFiles()
+                .StructuredCopyTo(context.ProjectDirectory, outputPath)
+                .RemoveAttribute(FileAttributes.ReadOnly);
```

Comment:
It's not clear what this is doing without digging deeper.

Is this copying content files and other such similar items? What if those files are not rooted in the ProjectDirectory (could that happen?)